src/azul/terraform.py

```diff
@@ -0,0 +1,29 @@
+import functools
+import itertools
+import json
+import os
+import subprocess
+from typing import (
+    Sequence,
+)
+
+from azul import (
+    require,
+)
+
+
+@functools.lru_cache
+def taggable_resources() -> Sequence[str]:
```

Comment:
This module is a bit anemic and I don't really see anything being added to it anytime soon. This should be defined in `deployment.py`.